src/components/general/Button.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+
+export default class Button extends Component {
+  render() {
+    const { type, text } = this.props;
```

Comment:
It seems slightly more idiomatic to make the text be passed in as children instead of as an explicit prop. In other words, the interface would look like this:

```jsx
<Button type="whatever">Hello world!</Button>
```

If you like that, I _think_ we'd just need to replace `text` with `children` here and update usages.

What do you think?

Reply:
Yeah that way is better / simpler. I was doing the same as the Link component which can be changed as well.